sdk/keyvault/azure-keyvault-secrets/azure/keyvault/secrets/client.py

```diff
@@ -169,16 +169,16 @@ def update_secret(
             error_map={404: ResourceNotFoundError},
             **kwargs
         )
-        return SecretAttributes._from_secret_bundle(bundle)  # pylint: disable=protected-access
+        return SecretProperties._from_secret_bundle(bundle)  # pylint: disable=protected-access
 
     @distributed_trace
-    def list_secrets(self, **kwargs):
-        # type: (**Any) -> ItemPaged[SecretAttributes]
+    def list_secret_properties(self, **kwargs):
```

Comment:
This should still be `list_secrets`. `list_secret_properties` suggests listing the properties of a single secret; actually it lists all secrets in the vault and happens to return a class we've named `SecretProperties`.

Reply:
switched back to list_secrets, thanks!